ionic2/src/components/type/type.component.ts

```diff
@@ -0,0 +1,12 @@
+import { Component, Input } from '@angular/core';
+
+import { TypeService } from '../../services/type.service';
+
+@Component({
+  selector: 'poke-type',
+  templateUrl: './type.component.html'
+})
+export class TypeComponent {
+  @Input() type: string;
+  constructor(private typeService: TypeService) { }
```

Comment:
Ah okay. But it did work somehow. Is ahead of time compilation disabled in dev mode?